src/frontend/metrics/generateMetricsReport.test.ts

```diff
@@ -1,29 +1,32 @@
 import * as path from 'node:path';
 import * as fs from 'node:fs';
-import generateMetricsReport from './generateMetricsReport';
+import generateMetricsReport, {addToSet} from './generateMetricsReport';
+import {generate} from '@mapeo/mock-data';
+import positionToCountries from './positionToCountries';
+import type {Observation} from '@mapeo/schema';
 
 type MetricsReportOptions = Parameters<typeof generateMetricsReport>[0];
 
 describe('generateMetricsReport', () => {
   const packageJson = readPackageJson();
+  const observations: ReadonlyArray<Observation | Record<string, never>> = [
+    ...generate('observation', {count: 10}),
+    {},
+  ];
+  const countries = new Set<string>();
+
+  for (const {lat, lon} of observations) {
+    if (typeof lat === 'number' && typeof lon === 'number') {
+      addToSet(countries, positionToCountries(lat, lon));
+    }
+  }
 
   const defaultOptions = {
     packageJson,
     os: 'android',
     osVersion: 123,
     screen: {width: 12, height: 34, ignoredValue: 56},
-    observations: [
-      // Middle of the Atlantic
-      {lat: 10, lon: -33},
-      // Mexico City
-      {lat: 19.419914, lon: -99.088059},
-      // Machias Seal Island, disputed territory
```

Comment:
Does the mock data include any disputed territories? I think that's useful to test.

Reply:
disputed in what sense? Politically? Everything is kinda random generated. But I can spread generated data with manual data (like the example here)